pyiron_contrib/nofiles/lammps.py

```diff
@@ -12,7 +12,16 @@ class LammpsInteractiveWithoutOutput(LammpsInteractive):
     def __init__(self, project, job_name):
         super(LammpsInteractiveWithoutOutput, self).__init__(project, job_name)
         self._interactive_disable_log_file = False
+        self._interactive_mpi_communicator = None
 
+    @property
+    def interactive_mpi_communicator(self):
+        return self._interactive_mpi_communicator
+
+    @interactive_mpi_communicator.setter
+    def interactive_mpi_communicator(self, comm):
+        self._interactive_mpi_communicator = comm
+        
```

Comment:
These lines are to necessary, once the new version of `pyiron_atomistics` is released